android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -45,20 +45,26 @@
 import com.smartdevicelink.managers.BaseSubManager;
 import com.smartdevicelink.managers.CompletionListener;
 import com.smartdevicelink.managers.StreamingStateMachine;
+import com.smartdevicelink.managers.video.resolution.AspectRatio;
```

Comment:
@sanhanamk, I don't think we should do it, otherwise the AspectRatio structure which was rolled back according to your suggestion will not work in this file